packages/design-tokens/CHANGELOG.md

```diff
@@ -11,11 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- **BREAKING:** Replaced Euclid Circular B with Centra No1 as the primary font family ([#499](https://github.com/MetaMask/metamask-design-system/pull/499)). See the [migration guide](./MIGRATION.md#from-version-510-to-600) for details.
+- **BREAKING:** Replaced Euclid Circular B with CentraNo1 as the primary font family ([#499](https://github.com/MetaMask/metamask-design-system/pull/499)). See the [migration guide](./MIGRATION.md#from-version-510-to-600) for details.
 
   - Removed `--font-family-euclid-circular-b` and `--font-family-roboto` CSS variables
-  - Changed `--font-family-sans` to use Centra No1 with updated fallback chain
-  - Updated font files from Euclid Circular B to Centra No1 (where 'Book' is the 400 weight variant)
+  - Changed `--font-family-sans` to use CentraNo1 with updated fallback chain
+  - Updated font files from Euclid Circular B to CentraNo1 (where 'Book' is the 400 weight variant)
```

Comment:
This is just a documentation update in the changelog but I think it will help to remove the spaces to prevent mistakes